File: app/src/main/java/com/highcom/passwordmemo/ui/list/BackgroundColorItem.java
package com.highcom.passwordmemo.ui.list;

public class BackgroundColorItem {
    private String mColorName;
    private Integer mColorCode;

    public BackgroundColorItem() {

    }

    public BackgroundColorItem(String colorName, Integer colorCode) {
        mColorName = colorName;
        mColorCode = colorCode;
    }

    public String getColorName() {
        return mColorName;
    }

    public Integer getColorCode() {
        return mColorCode;
    }
}

File: app/src/main/java/com/highcom/passwordmemo/ui/list/BackgroundColorItem.kt
package com.highcom.passwordmemo.ui.list

/**
 * 背景色設定用アイテム
 *
 * @constructor
 * 背景色設定用アイテムコンストラクタ
 *
 * @param colorName
 * @param colorCode
 */
class BackgroundColorItem(colorName: String?, colorCode: Int?) {
    /** 背景色名 */
    var colorName: String? = colorName
        private set
    /** 背景色コード */
    var colorCode: Int? = colorCode
        private set

}